benchmarks/multiplatform/settings.gradle.kts

```diff
@@ -6,16 +6,24 @@ pluginManagement {
         maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
         google()
     }
-
-    plugins {
-        val kotlinVersion = extra["kotlin.version"] as String
-        kotlin("multiplatform").version(kotlinVersion)
-        id("org.jetbrains.kotlin.plugin.compose").version(kotlinVersion)
-        val composeVersion = extra["compose.version"] as String
-        id("org.jetbrains.compose").version(composeVersion)
-    }
 }
 
 rootProject.name = "compose-benchmarks"
 
+dependencyResolutionManagement {
+    versionCatalogs{
+        create("libs") {
+            // Override Kotlin and Compose versions with properties
+            providers.run {
+                with(gradleProperty("kotlin.version")) {
+                    if (isPresent) version("kotlin", get())
+                }
+                with(gradleProperty("compose.version")) {
+                    if (isPresent) version("compose-multiplatform", get())
+                }
+            }
+        }
+    }
+}
+
```

Comment:
please, don't introduce it with the current PR. I know, the libs catalog is shiny cool feature but we have to keep our projects consistent with each other

Reply:
I've thought that you've been already using catalogues. For example, https://github.com/JetBrains/compose-multiplatform/tree/master/instrumented-test

Reply:
For me, it is ok to use version catalogs.